### PR TITLE
copy "Got a Question or Problem?" from angular/angular

### PR DESCRIPTION
**Pull request by fxck**

### CODE REVIEW

**Comment by jelbourn** [comment]
Looks good, though I know @StephenFluin will want a different tag listed. @StephenFluin, do we have up-to-date StackOVerflow tags?

**Comment by fxck** [comment]
I tested couple of tags, angular-material2 seems to be the most popular. Given that material2 repo name is going to stay(?), Shouldn't it be fine?

**Comment by fxck** [comment]
angular-material2 - 417 tags
material2 - 0 tags
material - 602 tags (too common anyway)
angular-material - 3176 tags (all of them materialjs)
angularjs-material - 41 tags

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,10 +17,17 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
 
 ## <a name="question"></a> Got a Question or Problem?
 
-If you have questions about how to *use* Angular Material, please direct them to the 
-[Google Group][material-group] discussion list or [StackOverflow][stackoverflow]. 
-Please note that Angular Material is still in very early development, and the team's capacity 
-to answer usage questions is limited. Community chat is also available on [Gitter][gitter].
+Please, do not open issues for the general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](https://stackoverflow.com/questions/tagged/angular-material2) where the questions should be tagged with tag `angular-material2`.
```
Comment (kara) [inline CONTRIBUTING.md]: `Please, do not` -> `Please do not`

`for the general support questions` -> `for general support questions`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,10 +17,17 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
...
+
+StackOverflow is a much better place to ask questions since:
+
+- there are thousands of people willing to help on StackOverflow
+- questions and answers stay available for public viewing so your question / answer might help someone else
+- StackOverflow's voting system assures that the best answers are prominently visible.
+
+To save your and our time we will be systematically closing all the issues that are requests for general support and redirecting people to StackOverflow.
```
Comment (kara) [inline CONTRIBUTING.md]: `To save your and our time we will be ...` -> `To save your and our time, we will be...`

**Comment by jelbourn** [comment]
LGTM, talked with Stephen and this tag is fine for now

**Pull request merged**